commons/types/account_test.go

```diff
@@ -83,7 +83,7 @@ func TestReadAccountFile(t *testing.T) {
 	defer testCleanAccountFile(t, name)
 	newAccount := readAccountFile(name)
 	existingAccount := readAccountFile(name)
-	if reflect.DeepEqual(newAccount, existingAccount) == false {
+	if newAccount.String() != existingAccount.String(){
```

Comment:
This test should not have had to change. If the existing "DeepEqual" check failed, then something is wrong in the "readAccountFile" method because multiple calls to the method MUST result in the same outcome when no other operations are involved.